### ISSUE TITLE
module 'lua-utf8' not found

### ISSUE THREAD

**Issue by konstantinosDMS** [starwing/luautf8#48]
Hi, I am using lua 5.4, i installed the luarocks and luarocks luautf8 package, but studio code searching the following locations for .so library
`Desktop/server/lua-apps/lua.lua:3118: module 'lua-utf8' not found:
        no field package.preload['lua-utf8']
        no file '/home/konstantinos/Desktop/server/lua-apps/lua-utf8.lua'
        no file '/home/konstantinos/Desktop/server/lua-apps/lua-utf8.lua'
        no file '/home/konstantinos/Desktop/server/lua-apps/lua-utf8.so'
        no file '/home/konstantinos/Desktop/server/lua-apps/lua-utf8.so'`
I required as docs say in my code. Any suggestion?

**Comment by alerque** [comment]
None of the paths mentioned there as getting searched look like anywhere `luarocks` may have installed anything. You need to adjust your `package.cpath` to include whatever system, user, or project location you installed to with LuaRocks.

**Comment by konstantinosDMS** [comment]
> None of the paths mentioned there as getting searched look like anywhere `luarocks` may have installed anything. You need to adjust your `package.cpath` to include whatever system, user, or project location you installed to with LuaRocks.

U mean to add cpath to my environment variables point to luarocks packages?

**Comment by alerque** [comment]
Yes. Or whatever configuration Studio Code offers you for setting such values.

This is not an issue with this project and only to do with the Lua search path in the environment you are trying to use it in not matching the one you installed to. As such this issue can be closed.
